courses/free-courses-en.md

```diff
@@ -67,6 +67,7 @@
 * [TypeScript](#typescript)
   * [Angular](#angular)
   * [Deno](#deno)
+  * [Electron](#electron)
```

Comment:
Sorry for annoy. As I post on issue (some editing later) https://github.com/EbookFoundation/free-programming-books/issues/5928#issuecomment-934353580 better in Javascript

https://github.com/EbookFoundation/free-programming-books/blob/c16072258eb5b0c8a36521003b8dee7ede9ce723/books/free-programming-books-zh.md#L48

Reply:
Ok noted.